### DISCUSSION TITLE
Deploy everything in Kubernetes

### DISCUSSION THREAD

**Discussion by govindkailas** [project-copacetic/copacetic#354] [Ideas]
Is it possible to deploy everything in Kubernetes, 
- buildkit
- Trivy
- copa 
and then expose copa as a service?

**Comment by sozercan** [comment]
@govindkailas Docker has a buildkit Kubernetes driver, so you should be able to utilize that today https://docs.docker.com/build/drivers/kubernetes/

Please let us know if you decide to build this, would be great to see!

**Reply by govindkailas** [reply]
sure thing, I shall post some updates here whenever I get this done!
